packages/logger/src/index.ts

```diff
@@ -108,21 +113,25 @@ export class ServiceLogger {
   /**
    * Converts the service log to logfmt and writes it to `this.filePath`
    * @param serviceLog Service log object
-   * @param logToConsole True to log to console in addition to file
    */
-  public log(serviceLog: ServiceLog, logToConsole?: boolean): void {
```

Comment:
I removed the `logToConsole` arg since it's not being used anywhere and was complicating some of the logic.  I figure if we have code that wants to make sure we always log to console it can always use the diagnostic logger directly at a higher log level.

Reply:
I think in debug mode the service loggers should always also log to console

Reply:
Otherwise it will be harder for us to see what is going on

Reply:
Yep, they still do that

Reply:
Oh ok I see you kept that anyway!